src/algebra/lie/basic.lean

```diff
@@ -774,7 +781,9 @@ instance : partial_order (lie_submodule R L M) :=
 { le := λ N N', ∀ ⦃x⦄, x ∈ N → x ∈ N', -- Overriding `le` like this gives a better defeq.
   ..partial_order.lift (coe : lie_submodule R L M → set M) coe_injective }
 
-lemma le_def (N N' : lie_submodule R L M) : N ≤ N' ↔ (N : set M) ⊆ N' := iff.rfl
+lemma le_def : N ≤ N' ↔ (N : set M) ⊆ N' := iff.rfl
+
+@[simp] lemma coe_le_coe_submodule : (N : submodule R M) ≤ N' ↔ N ≤ N' := iff.rfl
```

Comment:
This should definitely be a `norm_cast` lemma. Making the name more symmetrical, as in the suggestion, would be nice, if you don't feel it's too long.
```suggestion
@[simp, norm_cast] lemma coe_submodule_le_coe_submodule : (N : submodule R M) ≤ N' ↔ N ≤ N' := iff.rfl
```